canvas/biplot.py

```diff
@@ -0,0 +1,167 @@
+from __future__ import division
+import matplotlib
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+import numpy as np
+import pandas as pd
+from skbio.stats.composition import clr, closure, centralize,
+from skbio.stats.composition import multiplicative_replacement, perturb_inv
+from biom import load_table
+from functools import partial
+import math
+
+
+def make_biplot(sample_df,
+                feature_df,
+                sample_df_metadata=None,
+                feature_df_metadata=None,
+                sample_color_category=None,
+                feature_color_category=None,
+                sample_color_dict=None,
+                feature_color_dict=None,
+                eigenvalue=None):
```

Comment:
Also, weren't we supposed to be able to pass in an `matplotlib.axes` object into here?